src/components/Icons/LevelButton.js

```diff
@@ -0,0 +1,15 @@
+import { FaListOl } from "react-icons/fa";
+import { IconLink } from './styled';
+
+const LevelButton = () => {
+  return (
+    <IconLink
+      title="Level Select"
```

Comment:
href="#/levelselect"
We can go ahead and this attribute and location.

Reply:
same as above. Please update too
href="/levels"
I also made a mistake here of adding the hash symbol #.
Also made the error of not following what the url should be. Should be levels not levelselect

Reply:
to="/levels"
just levels now would be correct. My mistake on telling you otherwise.